db/migrate/20200114095428_create_whitehall_migration_timeline_entries.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateWhitehallMigrationTimelineEntries < ActiveRecord::Migration[6.0]
+  def change
+    create_table :whitehall_migration_timeline_entries do |t|
+      t.string "entry_type"
+      t.text "body"
+      t.timestamps
```

Comment:
I think as we'd not expect this record to ever be updated that we'd just need a created_at here, see https://github.com/alphagov/content-publisher/blob/master/db/migrate/20190626155821_create_access_limits.rb#L6 for past approach

Reply:
Done